components/rickroll/leeway.Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License-AGPL.txt in the project root for license information.
+
+FROM scratch
+
+COPY bin /ide/bin
+COPY startup.sh /ide/
+COPY supervisor-ide-config.json  /ide/
+COPY index.html  /ide/
+
+RUN chmod -R ugo+x /ide/bin
+RUN chmod -R ugo+x /ide/startup.sh
+
+ENV GITPOD_ENV_APPEND_PATH /ide/bin:
```

Comment:
ENVs can be removed, they are used to mutate container ENVs to add code cli and use as an editor.